File: web3rpc/rpc-specs/code-samples/python/debug/logging/verbosityByName.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

name = "API"
level = 1

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.verbosity_by_name(name, level)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/logging/verbosityByName.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const name = "API";
    const level = 1;

    sdk.debug.verbosityByName(name, level, {}, (err, data, response) => {
        console.log(data);
    });
}
)()